### DISCUSSION TITLE
Empty space after root node

### DISCUSSION THREAD

**Discussion by psmolak** [nvim-neo-tree/neo-tree.nvim#380] [Q&A] [answered]
Hi!

I was wondering if it's possible to achieve following effect where the first line displays root path, then it's followed by one empty line and then the rest of the tree is rendered?

<img width="371" alt="image" src="https://user-images.githubusercontent.com/10662415/174277534-7357e0f5-ca04-414e-bd5b-2b562118bc9e.png">

**Comment by cseickel** [comment, marked as answer]
There's no simple way to do this, but you could probably use the `after_render` event to inject a blank node as the first child of the root node. That root node could be of type `message` which would ensure that it is ignored by mappings and other internal logic.

It's not a simple task and there are no examples. I might be willing to code it up as an example if there were a lot of people looking for this. I'm certainly available for questions if you wanted to take a shot at it yourself.

**Reply by nyngwang** [reply]
WIll be a great feature if this can be achieved in simple way.
